### ISSUE TITLE
i think it's better to package your API project and send to Maven

### ISSUE THREAD

**Issue by saselovejulie** [AdminAnticaptcha/anticaptcha-java#2]
live in china, download this project need a lot of time. if i can use your API through Maven it should be great.

**Comment by MrJavaNoHome** [comment]
Checkout https://github.com/MrJavaNoHome/anticaptcha-java

**Comment by dore51** [comment]
@MrJavaNoHome The problem is that your project is not maintained and is behind this project, can you release an update?

**Comment by MarioCodarin** [comment]
@MrJavaNoHome yes would be great. for me error 503 when trying install from your repo

**Comment by MrJavaNoHome** [comment]
@dore51 I've updated it.
@MarioCodarin Probably temp unavailability of mvn repository server.

**Comment by MarioCodarin** [comment]
Thank you so much!
